package.json

```diff
@@ -2,26 +2,29 @@
 	"name": "@fandom/jwplayer-fandom",
 	"version": "2.8.1",
 	"description": "JWPlayer for Fandom",
+	"engines": {
+		"node": ">=18.0.0"
+	},
 	"exports": {
-		".": "./main.js",
-		"./DesktopArticleVideoLoader": "./DesktopArticleVideoLoader.js",
-		"./MobileArticleVideoLoader": "./MobileArticleVideoLoader.js",
-		"./CanonicalVideoLoader": "./CanonicalVideoLoader.js",
-		"./RedVentureVideoPlayer": "./RedVentureVideoPlayer.js"
+		".": "./dist/main.js",
+		"./DesktopArticleVideoLoader": "./dist/DesktopArticleVideoLoader.js",
+		"./MobileArticleVideoLoader": "./dist/MobileArticleVideoLoader.js",
+		"./CanonicalVideoLoader": "./dist/CanonicalVideoLoader.js",
+		"./RedVentureVideoPlayer": "./dist/RedVentureVideoPlayer.js"
 	},
-	"types": "main.d.ts",
+	"files": ["dist"],
```

Comment:
Instead of publishing `dist` directory with copied `package.json` now we will be publishing from the root directory of the package but just the `dist` dir. To make it work I needed to update `exports` paths to contain `dist` dir.